test/admin/sync.test.js

```diff
@@ -225,40 +225,6 @@ describe('Sync', () => {
       expect(deployResult.status).toBe('created');
     });
 
-
-    it('can use the allow_destructive_changes param to do a destructive rule create', async() => {
-      const syncService = await createSampleMongodbSyncService(services);
-
-      const schemaInvalidatingRule = {
-        database: 'db',
-        collection: 'coll',
-        config: {
-          schema: {
-            title: 'double',
-            properties: {
-              _id: { bsonType: 'objectId' },
-              key: { bsonType: 'string' },
-              obj: {
-                bsonType: 'object',
-                title: 'double'
-              }
-            }
-          }
-        }
-      };
-
-      let stitchError;
-      try {
-        await addRuleToMongodbService(services, syncService, schemaInvalidatingRule );
-      } catch (e) {
-        stitchError = e;
-      }
-      expect(stitchError.code).toBe('InvalidSyncSchema');
-
-      const createResult = await addRuleToMongodbService(services, syncService, schemaInvalidatingRule, { allow_destructive_changes: true });
-      expect(createResult).toBeTruthy();
-    });
```

Comment:
Things have changed in the server so you can no longer have a destructive change on create. Newly created invalid schemas are not destructive to sync as they are now ignored by sync entirely.

Reply:
Perhaps you should keep the test in and just remove the piece that expects an error and tries to bypass it. That way this confirms that you can add an sync-incompatible schema without experiencing an error.